src/pages/Order/Order.js

```diff
@@ -0,0 +1,185 @@
+      {modalOpen && (
+        <Modal
+          modalOpen={() => setModalOpen(!modalOpen)}
+          setModalOpen={setModalOpen}
+        />
+      )}
+
+      <div className="section-top">
+        <div className="logo" />
+      </div>
+      <div className="section-main">
+        <div className="section-text">
+          <div className="section-header">
+            <div className="step-header">배송과 픽업 방법</div>
+          </div>
+          <div className="delivery-address">배송지:</div>
+          <div className="address-text">테헤란로 427</div>
+          <div onClick={() => navigate('/cart')} className="fix-address">
+            수정
+          </div>
+          <div className="delivery-type-header">배송 방법</div>
+          <div className={`delivery-option-box ${delivery && `active`}`}>
+            <div className="title-wrap">
+              <div className="delivery-option-title">무료 배송</div>
+              <div className="delivery-option-title">₩ 0</div>
+            </div>
+            <div className="delivery-time">1-7일</div>
+            <div className="delivery-time">배송일 - 가장 빠른 날짜</div>
+            <div className="delivery-time">
+              {new Date().toLocaleDateString('en-US', {
+                year: 'numeric',
+                month: '2-digit',
+                day: '2-digit',
+              })}
+            </div>
+            <div className="no-contact-box">
+              <div className="delivery-option-title">비대면 배송</div>
+              <div className="checkbox-container">
+                <input
+                  onClick={() =>
+                    setRequirements({ ...requirements, delivery: !delivery })
+                  }
+                  type="checkbox"
+                  className="checkbox"
+                />
+                <div className="checkbox-label">제품을 현관 앞에 놔두기</div>
+              </div>
+            </div>
+          </div>
+          <div className={!delivery && 'inactive'}>
+            <div className="section-header">
+              <div className="step-header">결제</div>
+            </div>
+            <div className="payment-options">
+              <div className="payment-text">위케아 포인트</div>
+              <div className="option-thumbs-1" />
+              <div className="option-thumbs-2" />
+            </div>
+            <div className="cost">
+              <div className="total">총 주문금액</div>
+              <div className="cost-num">₩ 79,000</div>
+            </div>
+            <div className="terms-wrapper">
+              <input
+                onClick={() =>
+                  setRequirements({ ...requirements, terms: !terms })
+                }
+                type="checkbox"
+                className="terms"
+              />
+              <div>이용 약관을 읽었으며 동의합니다.</div>
+            </div>
+            <div
+              onClick={() => setModalOpen(!modalOpen)}
+              className={`btn-purchase ${
+                (delivery === false || terms === false) && `disabled`
```

Comment:
```suggestion
                fomNotComplete && `disabled`
```
위에 있는 값을 이용하셔도 될 것 같네요!